src/Common/tests/System/Diagnostics/RemoteExecutorConsoleApp/RemoteExecutorConsoleApp.cs

```diff
@@ -59,12 +63,27 @@ private static int Main(string[] args)
             }
             catch (Exception exc)
             {
-                Console.Error.WriteLine("Exception from RemoteExecutorConsoleApp({0}):", string.Join(", ", args));
-                Console.Error.WriteLine("Assembly: {0}", a);
-                Console.Error.WriteLine("Type: {0}", t);
-                Console.Error.WriteLine("Method: {0}", mi);
-                Console.Error.WriteLine("Exception: {0}", exc);
-                throw exc;
+                if (exc is TargetInvocationException && exc.InnerException != null)
+                    exc = exc.InnerException;
```

Comment:
Ok. I'd personally have left it exactly so that no potential info is lost (including that it actually did come from the invoke), but not a big deal. Thanks.